app/shortcuts/shortcuts.tsx

```diff
@@ -141,9 +149,7 @@ export class Shortcuts {
     let shortcut = new Shortcut(keyCombo, action);
     for (let otherShortcut of this.shortcuts.values()) {
       if (shortcut.collidesWith(otherShortcut)) {
-        console.warn("Duplicate keyboard shortcut registered: " + shortcut.keyCombo);
-        // TODO(iain): throw an error here once bugs are bashed.
-        // throw new Error("Duplicate keyboard shortcut registered: " + shortcut.keyCombo);
+        throw new Error("Duplicate keyboard shortcut registered: " + shortcut.keyCombo);
```

Comment:
I'd slightly prefer keeping this a `console.warn` and us figuring a way to do some sort of client side error reporting. Don't think crashing the page does a ton of good without any way of displaying to the user what the error is (or reporting it back to us).

Reply:
sg, switched back to console.warn() with a TODO